README.md

```diff
@@ -0,0 +1,8 @@
+# googleAPIClone
+
+# Upon Startup
+
+Upon start, please ensure that you have installed all dependencies with ```npm install```
```

Comment:
Could be more specific. Need to run `npm install` in both client and server folders independently.